install/create-kafka-topics.sh

```diff
@@ -1,12 +1,24 @@
 echo "${_group}Creating additional Kafka topics ..."
 
-# NOTE: This step relies on `kafka` being available from the previous `snuba-api bootstrap` step
+$dc up --no-build --no-recreate kafka
+
+while [ true ]
+do
+    $kafka_healthy=$($dc ps kafka | grep 'healthy')
+    if [ ! -z "$kafka_healthy" ]; then
+      break
+    fi
+
+    echo "Kafka container is not healthy, waiting for 30 seconds. If this took too long, abort the installation process, and check your Kafka configuration"
+    sleep 30s
+done
```

Comment:
Protip, you can do something like this: https://github.com/getsentry/self-hosted/blob/7143d887430ebfa624df75d18448a49df6c908c5/_integration-test/run.sh#L47

```suggestion
echo "Waiting for kafka container to be healthy for up to 30 seconds."
timeout 30 bash -c 'until $($dc ps kafka | grep -q 'healthy'); do printf '.'; sleep 0.5; done'
```

Reply:
Well, then let's make it 300s? :D

Reply:
So... Applying the change with at least 300s? Maybe it's better to even raise it a bit further, just in case?

Reply:
To be honest, I'm not 100% sure. I don't use Kafka anymore on my Sentry instance, but for a minimum 4 core CPU, Kafka is rather problematic to boot up. I think it's okay to stay as it is (rather than changing to Burak's recommendation).

Reply:
@BYK What's your opinionen? Would be nice to get this merged

Reply:
Okay let's settle this at 150s :laughing:

Reply:
I think the reason CI is failing is because you may need to remove the $ in front of $kafka_healthy

Reply:
@hubertdeng123 can you just directly commit it?